docs/requirements.md

```diff
@@ -1,51 +1,140 @@
 # Requirements and Recommendations for the Underlying Databases of ScalarDB
 
-This document explains the requirements and recommendations in the underlying databases of ScalarDB to make ScalarDB applications work correctly.
+This document explains the requirements and recommendations in the underlying databases of ScalarDB to make ScalarDB applications work correctly and efficiently.
 
-## Common requirements
+## Requirements
 
-This section describes common requirements for the underlying databases when using ScalarDB.
+ScalarDB requires each underlying database to provide certain capabilities for running transactions and analytics on the databases. This document first explains the general requirements and then explains how to configure each database to achieve the requirements.
 
-### Privileges to access the underlying databases
+### General requirements
 
-ScalarDB operates the underlying databases not only for CRUD operations but also for performing operations like creating or altering schemas, tables, or indexes. Thus, ScalarDB basically requires a fully privileged account to access the underlying databases.
+#### Transactions
+{:.no_toc}
+ScalarDB requires each underlying database to provide at least the following capabilities for running transactions on the databases:
+- Linearizable read and conditional mutations (write and delete) on a single database record.
+- Durability of written database records.
+- Ability to store arbitrary data besides application data in each database record.
 
-## Cassandra or Cassandra-compatible database requirements
+{% capture notice--info %}
+**Note**
+
+You need to have database accounts that have enough privileges to access the databases thorough ScalarDB since ScalarDB runs on the underlying databases not only for CRUD operations but also for performing operations like creating or altering schemas, tables, or indexes. ScalarDB basically requires a fully privileged account to access the underlying databases.
```

Comment:
```suggestion
You need to have database accounts that have enough privileges to access the databases through ScalarDB since ScalarDB runs on the underlying databases not only for CRUD operations but also for performing operations like creating or altering schemas, tables, or indexes. ScalarDB basically requires a fully privileged account to access the underlying databases.
```